test/e2e/kubectl.go

```diff
@@ -375,6 +375,48 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 		})
 	})
 
+	framework.KubeDescribe("kubectl plugins", func() {
```

Comment:
If it makes sense to have this test in a place other than e2e, we should move it. Can't review now